gocron.go

```diff
@@ -115,20 +115,22 @@ func (j *Job) run() (result []reflect.Value, err error) {
 		}()
 	}
 
-	f := reflect.ValueOf(j.funcs[j.jobFunc])
-	params := j.fparams[j.jobFunc]
+	result, err = callJobFuncWithParams(j.funcs[j.jobFunc], j.fparams[j.jobFunc])
```

Comment:
should we ignore the error in this case?

Reply:
Yes, seems legit. Whoever is calling the function is expected to know what the args being passed are.